numba/typeconv/typeconv.cpp

```diff
@@ -184,34 +189,6 @@ int TypeManager::_selectOverload(Type sig[], Type ovsigs[], int &selected,
     return matchcount;
 }
 
-// ----- Ratings -----
-Rating::Rating() :promote(0), safe_convert(0), unsafe_convert(0) { }
-
-void Rating::bad() {
```

Comment:
No, the Rating type is still used, it's just moved around :)
(I was assuming that maybe the compiler wasn't able to inline something that's defined after its use, but I may be wrong about that)